design/help_overview.md

```diff
@@ -212,9 +212,51 @@ pywbemcli server profiles --help
 b'Usage: pywbemcli server profiles [COMMAND-OPTIONS]\n\n  Display profiles on the current WBEM Server.\n\n  This display may be filtered by the optional organization and profile name\n  options\n\nOptions:\n  -o, --organization <org name>   Filter by the defined organization. (ex. -o\n                                  DMTF\n  -n, --profilename <profile name>\n                                  Filter by the profile name. (ex. -n Array\n  --help                          Show this message and exit.\n'
```

Comment:
This results in a very long line, not only in the source file but also in the rendered output. Split accordingly.